terraform/browser/browser.tf.json.template.py

```diff
@@ -78,6 +136,40 @@ def emit():
             }
         },
         'resource': {
+            'aws_cloudfront_response_headers_policy': {
```

Comment:
Please follow precedent by putting CF support resources along with the other CF support resources. Within those resource definitions, find a suitable place. If you can't find one or if you are in doubt, append, don't prepend.

Reply:
Note, due to the three commits in this PR that touch this resource, a fixup commit was not created for the relocation of the resource, rather the commits were amended.